lib/command/init.js

```diff
@@ -80,7 +87,19 @@ module.exports = function (initPath) {
     return;
   }
 
+  const typeScriptconfigFile = path.join(testsPath, 'codecept.conf.ts');
+  if (fileExists(typeScriptconfigFile)) {
+    error(`Config is already created at ${typeScriptconfigFile}`);
+    return;
+  }
+
   inquirer.prompt([
+    {
+      name: 'typescript',
+      type: 'confirm',
+      choices: ['No', 'Yes'],
```

Comment:
did you try this actually?
does confirm option actually needs choices?

Reply:
Ah no. Will look closely then.